PSReadLine/ReadLine.vi.cs

```diff
@@ -282,14 +279,13 @@ public static void DeleteWord(ConsoleKeyInfo? key = null, object arg = null)
 
         private static void DeleteToEndPoint(object arg, int endPoint, Action<ConsoleKeyInfo?, object> instigator)
         {
-            _singleton.SaveToClipboard(_singleton._current, endPoint - _singleton._current);
-            _singleton.SaveEditItem(EditItemDelete.Create(
-                _clipboard,
+            _singleton.RemoveTextToClipboard(
                 _singleton._current,
+                endPoint - _singleton._current,
                 instigator,
                 arg
-                ));
-            _singleton._buffer.Remove(_singleton._current, endPoint - _singleton._current);
+                );

```

Comment:
Minor: move the end parenthese to the end of the last argument.

```suggestion
                arg);
```